source/CardGames.py

```diff
@@ -355,8 +446,21 @@ def winnnerCheck(self, house, player):
       print("It's a push! You both have blackjack.")
       betting_box.collect(blackjackOutcome)
       print(f"You now have ${player_name.money}")
-      
+   
     else:
+    
```

Comment:
This should be a method under the Betting_box class then called with one line in the blackjack() function.